types/schemas/prototypeApplication/index.ts

```diff
@@ -49,10 +50,17 @@ export type LandDrainageConsent = ApplicationSpecification<
   'landDrainageConsent',
   LandDrainageConsentApplicationType
 >;
+export type WTT = ApplicationSpecification<'wtt', WTTApplicationType>;
 // TODO: All the rest!
 
 /**
  * @title PrototypeApplication
  * @description The (prototype) root specification for a planning application in England generated by a digital planning service
  */
-export type PrototypeApplication = LDC | PA | PP | Listed | LandDrainageConsent;
+export type PrototypeApplication =
+  | LDC
+  | PA
+  | PP
+  | Listed
+  | LandDrainageConsent
+  | WTT;
```

Comment:
Once merged to `main`, we should automatically see the new `WTT` value as an available option in the dropdown selection here :star: 
https://theopensystemslab.github.io/digital-planning-data-schemas-docs/
![Screenshot from 2025-02-04 17-29-51](https://github.com/user-attachments/assets/12be2b03-938d-4331-b2aa-cfd094b9657c)